src/containers/current-item.js

```diff
@@ -1,35 +1,39 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
-class CurrentItem extends React.Component{
-    render(){
-        console.log("test")
-        if(!this.props.selectedVideo && this.props.videoList){
-            return <div>waiting for video selection</div>;
-	    }
+class CurrentItem extends React.Component {
+  render() {
```

Comment:
add before render()

const {selectedVideo, videoList} = this.props;

and now you can use the props